packages/lab/src/drawer/Drawer.css

```diff
@@ -8,14 +8,11 @@
   overflow: auto;
   padding: var(--salt-spacing-300);
   box-shadow: var(--salt-overlayable-shadow-modal);
-  z-index: calc(var(--salt-zIndex-appHeader) - 1);
+  z-index: var(--salt-zIndex-drawer);
   position: fixed;
   top: 0;
-}
-
-.saltDrawer-overlay {
-  background: var(--salt-overlayable-background);
-  z-index: var(--salt-zIndex-drawer);
+  min-width: calc(2 * var(--salt-spacing-300) + var(--salt-size-base));
+  min-height: calc(2 * var(--salt-spacing-300) + var(--salt-size-base));
```

Comment:
Is min-width/min-height applicable on all positions?

Reply:
it was until design review on Tues, agreement is to have them all removed